index.html

```diff
@@ -108,74 +108,74 @@ <h4 class="pro">Profesional Art Printing Data</h4>
-        <button class="pro-btn" type="button">See Project</button> 
+        <button class="pro-btn" onclick="appear()" type="button">See Project</button> 
       </div>  
       <div>
         <div  class="header-pro grid-3">
           <h4 class="pro">Profesional Art Printing Data</h4>
           <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
           been the industry's standard</p>
-          <ul class="pro-ul">
+          <ul class="pro-ul" onclick="appear()">
             <li class="pro-html">html</li>
             <li class="pro-boot">bootsrap</li>
             <li class="pro-rubby">Ruby</li>
           </ul>
         </div>
-        <button class="pro-btn" type="button">See Project</button> 
+        <button class="pro-btn" onclick="appear()" type="button">See Project</button> 
       </div>  
       <div>
         <div  class="header-pro grid-1">
           <h4 class="pro">Profesional Art Printing Data</h4>
           <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
           been the industry's standard</p>
-          <ul class="pro-ul">
+          <ul class="pro-ul" onclick="appear()">
             <li class="pro-html">html</li>
             <li class="pro-boot">bootsrap</li>
             <li class="pro-rubby">Ruby</li>
           </ul>
         </div>
-        <button class="pro-btn" type="button">See Project</button> 
+        <button class="pro-btn" onclick="appear()" type="button">See Project</button> 
       </div>  
       <div>
         <div  class="header-pro grid-2">
           <h4 class="pro">Profesional Art Printing Data</h4>
           <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
           been the industry's standard</p>
-          <ul class="pro-ul">
+          <ul class="pro-ul" onclick="appear()">
             <li class="pro-html">html</li>
             <li class="pro-boot">bootsrap</li>
             <li class="pro-rubby">Ruby</li>
           </ul>
         </div>
-        <button class="pro-btn" type="button">See Project</button> 
+        <button class="pro-btn" onclick="appear()" type="button">See Project</button> 
       </div>  
       <div>
         <div  class="header-pro grid-3">
           <h4 class="pro">Profesional Art Printing Data</h4>
           <p class="pro-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has
           been the industry's standard</p>
-          <ul class="pro-ul">
+          <ul class="pro-ul" onclick="appear()">
             <li class="pro-html">html</li>
             <li class="pro-boot">bootsrap</li>
             <li class="pro-rubby">Ruby</li>
           </ul>
         </div>
-        <button class="pro-btn" type="button">See Project</button> 
+        <button class="pro-btn" onclick="appear()" type="button">See Project</button> 
       </div>
     </div>      
```

Comment:
- [ ] According to the requirements, You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads. so Kindly remove all the HTML code of the project section, and create this section dynamically using JS 💯👌

![11](https://user-images.githubusercontent.com/45231276/189283972-a5d47dc1-ccaa-494f-ab6a-fca2e94da86c.PNG)